CI: e2e tests broken — engineerd/setup-kind@v0.6.2 tag points to commit missing dist/

## Summary

All e2e tests on `run-e2e-test-kind` are failing with:

```
##[error]File not found: '/home/runner/work/_actions/engineerd/setup-kind/v0.6.2/dist/main/index.js'
```

The `v0.6.2` tag on `engineerd/setup-kind` now resolves to a commit that does not contain the compiled `dist/` directory required by `action.yml`.

## Evidence

The tag was updated to point to a different commit between ~10:00 and ~20:00 UTC on June 12, 2026:

| Run | Time (UTC) | Resolved SHA | Has `dist/`? | Result |
|-----|-----------|--------------|-------------|--------|
| [27409262572](https://github.com/vmware-tanzu/velero/actions/runs/27409262572) | 10:16 | `ecfad617` ("chore(release): release v0.6.2") | Yes | Pass |
| [27357453082](https://github.com/vmware-tanzu/velero/actions/runs/27357453082) | 20:11 | `71e45b96` ("Merge pull request #256 from RoadRunnr/fix/json5") | No | Fail |

The `engineerd/setup-kind` repo has had no merges since October 2024 and has 18 open PRs (oldest from 2022). An unexpected tag update on a dormant repo may warrant caution.

## Affected workflow

`.github/workflows/e2e-test-kind.yaml:136`:
```yaml
- uses: engineerd/setup-kind@v0.6.2
```

## Suggested fix

Pin to the known-good commit SHA:

```yaml
- uses: engineerd/setup-kind@ecfad61750951586a9ef973db567df1d28671bdc  # v0.6.2
```

Or consider migrating to an actively maintained alternative.

--- a/.github/workflows/e2e-test-kind.yaml
+++ b/.github/workflows/e2e-test-kind.yaml
@@ -137,6 +137,7 @@ jobs:
           docker run -d --rm -p 9000:9000 -e "MINIO_ROOT_USER=minio" -e "MINIO_ROOT_PASSWORD=minio123" -e "MINIO_DEFAULT_BUCKETS=bucket,additional-bucket" bitnami/minio:local
       - uses: helm/kind-action@v1
         with:
+          cluster_name: "kind"
           version: "v0.32.0"
           node_image: "kindest/node:v${{ matrix.k8s }}"
       - name: Fetch built CLI

--- a/.github/workflows/e2e-test-kind.yaml
+++ b/.github/workflows/e2e-test-kind.yaml
@@ -1,4 +1,6 @@
 name: "Run the E2E test on kind"
+env:
+  FORCE_JAVASCRIPT_ACTIONS_TO_NODE24: true
 on:
   push:
   pull_request:
@@ -133,11 +135,10 @@ jobs:
       - name: Install MinIO
         run: |
           docker run -d --rm -p 9000:9000 -e "MINIO_ROOT_USER=minio" -e "MINIO_ROOT_PASSWORD=minio123" -e "MINIO_DEFAULT_BUCKETS=bucket,additional-bucket" bitnami/minio:local
-      - uses: engineerd/setup-kind@v0.6.2
+      - uses: helm/kind-action@v1
         with:
-          skipClusterLogsExport: true
           version: "v0.32.0"
-          image: "kindest/node:v${{ matrix.k8s }}"
+          node_image: "kindest/node:v${{ matrix.k8s }}"
       - name: Fetch built CLI
         id: cli-cache
         uses: actions/cache@v4

--- a/.github/workflows/e2e-test-kind.yaml
+++ b/.github/workflows/e2e-test-kind.yaml
@@ -60,7 +60,7 @@ jobs:
       - name: Check Bitnami MinIO Dockerfile version
         id: minio-version
         run: |
-          DOCKERFILE_SHA=$(curl -s https://api.github.com/repos/bitnami/containers/commits?path=bitnami/minio/2025/debian-12/Dockerfile\&per_page=1 | jq -r '.[0].sha')
+          DOCKERFILE_SHA=$(curl -s https://api.github.com/repos/bitnami/containers/commits?path=bitnami/minio/2026/debian-12/Dockerfile\&per_page=1 | jq -r '.[0].sha')
           echo "dockerfile_sha=${DOCKERFILE_SHA}" >> $GITHUB_OUTPUT
       - name: Cache MinIO Image
         uses: actions/cache@v4

